src_c/math.c

```diff
@@ -2059,6 +2059,15 @@ vector2_rotate_ip_rad(pgVector *self, PyObject *angleObject)
     Py_RETURN_NONE;
 }
 
+static PyObject *
+vector2_rotate_ip_rad(pgVector *self, PyObject *angleObject)
+{
+    if (PyErr_WarnEx(PyExc_DeprecationWarning, "vector2_rotate_rad_ip() now has all the functionality of vector2_rotate_ip_rad(), so vector2_rotate_ip_rad() will be deprecated in pygame 2.2", 1) == -1) {
```

Comment:
the version number here needs updating. And similar changes needed in other functions